### ISSUE TITLE
Error in with useThemeManager() in version 0.0.29

### ISSUE THREAD

**Issue by jamesj2** [kaandesu/vue-daisyui-theme-manager#5]
Just trying out your package and ran across this the following error.

Error: 
```
console-breadcrumbs.js:33 TypeError: Cannot read properties of undefined (reading 'theme')
    at h (vue-daisyui-theme-manager.js:21:14)
    at n (vue-daisyui-theme-manager.js:23:3)
    at setup (Navigation.vue:115:16)
    at callWithErrorHandling (runtime-core.esm-bundler.js:195:19)
    at setupStatefulComponent (runtime-core.esm-bundler.js:7612:25)
    at setupComponent (runtime-core.esm-bundler.js:7573:36)
    at mountComponent (runtime-core.esm-bundler.js:5901:7)
    at processComponent (runtime-core.esm-bundler.js:5867:9)
    at patch (runtime-core.esm-bundler.js:5335:11)
    at ReactiveEffect.componentUpdateFn [as fn] (runtime-core.esm-bundler.js:6091:9)
```

Tracked it down to:
```js 
d("default")
```
should be 
```js 
 d({theme:"default"})
```

Context:
```js
const l = d(
  window.matchMedia && window.matchMedia("(prefers-color-scheme: dark)").matches
), e = i({
  light: "light",
  dark: "dark",
  watchSystemTheme: !1
}), m = d("default"), u = d(!1), n = () => {
```

**Comment by kaandesu** [comment]
Hey there, I'll check the issue. Can you provide additional context, on how you used it in your code (to reproduce it)?

**Comment by jamesj2** [comment]
A reproduction can be found here: https://stackblitz.com/edit/vitejs-vite-1tbhjx.

**Comment by kaandesu** [comment]
Thanks for providing it. I reproduced from scratch and had the same issue.
In your code, you started the project with regular JavaScript. But in the code you provided the App.vue is using
` <script setup lang="ts> ..` so it is in typescript mode but your project is not setup for typescript.
So all the type annotations will not work.
But, if you create a new vite project with typescript, it should be all fine. I was able to get it working doing the same steps with just selecting typescript at the very start.

If you don't want to use any typescript, you can try clearing the type annotations in the App.vue and just use it with <script setup> .... It might not work, if that's the case feel free to open another issue for javascript support and i'll see what I can do.

_TL;DR: Workinging with Vue with Typescript, not Javascript_

**Comment by jamesj2** [comment]
I'm getting the error in the browser console.   I tried what you suggested [here](https://stackblitz.com/edit/vitejs-vite-kygcrt) but no luck. 

![image](https://github.com/kaandesu/vue-daisyui-theme-manager/assets/1355847/eaf48409-8f07-4bc2-ad67-b254a8abdd34)

**Comment by kaandesu** [comment]
can you create a `theme-manager-config.ts` in the root folder and add the following: (normally when you npm install the package, this is created automatically with 'postinstall' but i see thats not the case in stackblitz)
```ts
// add the names of the themes you want to use here
// warning: you need to specify them in tailwind.config.js as well
// DO NOT REMOVE: 'default', 'light', 'dark'
export default[
  'default',
  'light',
  'dark',
  'storm',
  'breeze',
  'cupcake',
  'bumblebee',
  'emerald',
  'corporate',
  'synthwave',
  'retro',
  'cyberpunk',
  'valentine',
  'halloween',
  'garden',
  'forest',
  'aqua',
  'lofi',
  'pastel',
  'fantasy',
  'wireframe',
  'black',
  'luxury',
  'dracula',
  'cmyk',
  'autumn',
  'business',
  'acid',
  'lemonade',
  'night',
  'coffee',
  'winter',
] as const;

```

**Comment by kaandesu** [comment]
Also don't forget to add the daisyUI themes to the tailwind config file
